tests/net/all/prj.conf

```diff
@@ -106,8 +106,6 @@ CONFIG_NET_CONTEXT_SYNC_RECV=y
 CONFIG_NET_CONTEXT_CHECK=y
 
 # SLIP
-CONFIG_NET_SLIP_TUN=n
-CONFIG_NET_SLIP_TAP=y
```

Comment:
TUN is gone, if you add TAP here, it will fail for boards that have a real network device, TAP will be added automatically for QEMU targets.